src/freenas/usr/local/bin/truenas-initrd.py

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env python3
+import argparse
 import contextlib
 import json
+import libzfs
```

Comment:
please revert the import changes to libzfs and pyudev.

Reply:
@yocalebo doing that..could you clarify please why that is though so that i can be careful in the future too?

Reply:
builtin libraries go up top, 3rd party libraries go next, then all our libraries get imported.